library/src/main/java/com/schibsted/spain/barista/interaction/BaristaKeyboardInteractions.kt

```diff
@@ -0,0 +1,24 @@
+package com.schibsted.spain.barista.interaction
+
+import android.support.annotation.IdRes
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.action.ViewActions.closeSoftKeyboard
+import android.support.test.espresso.action.ViewActions.pressImeActionButton
+import android.support.test.espresso.matcher.ViewMatchers.isRoot
+import android.support.test.espresso.matcher.ViewMatchers.withId
+import com.schibsted.spain.barista.interaction.BaristaScrollInteractions.safelyScrollTo
+import com.schibsted.spain.barista.internal.matcher.DisplayedMatchers.displayedAnd
+
+object BaristaKeyboardInteractions {
+
+    @JvmStatic
+    fun close() {
```

Comment:
Suggestion: Thinking about api usage, if people want to statically import Barista methods to avoid the long class prefix, this should be called `closeKeyboard()` to better predict what is it about.

Reply:
👍